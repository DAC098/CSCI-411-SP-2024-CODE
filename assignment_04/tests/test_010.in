4
3 1 1
cttta aacgatat
hdvngitvewlqogsrmoktahvsliybwnixltj egfmlwnnusqvcjnzerylzxgmcfc
ogihbbbjcbcwirifywtodhq ycddhyuyvlvwj
catagagctcagccg agccccg
