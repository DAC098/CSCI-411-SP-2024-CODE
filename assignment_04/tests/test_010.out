__cttta_
aacgatat
11
hdvngitvewlqogsrmoktahvsliybwnixltj
___eg_fmlw_nnus___qvcjnzerylzxgmcfc
31
ogihbbbjcbcwirifywtodhq
_________ycddhyuy_vlvwj
21
catagagctcagccg
_____agc_c__ccg
8
